src/LinuxTracing/SwitchesStatesNamesVisitor.cpp

```diff
@@ -112,11 +113,12 @@ void SwitchesStatesNamesVisitor::Visit(uint64_t event_timestamp,
                            event_data.was_created_by_pid);
 }
 
-void SwitchesStatesNamesVisitor::Visit(uint64_t event_timestamp,
-                                       const SchedSwitchPerfEventData& event_data) {
-  // Note that context switches with tid 0 are associated with idle CPU, so we never consider them.
-
+template <typename SchedSwitchEventData>
```

Comment:
Not a fan of the type name, just dropping "Perf" from `SchedSwitchPerfEventData`. I would prefer `SchedSwitchPerfEventDataT`.

Reply:
Done.